commons-civil/src/main/java/uk/gov/hmcts/reform/sendgrid/EmailAttachment.java

```diff
@@ -1,4 +1,4 @@
-package uk.gov.hmcts.reform.civil.sendgrid;
+package uk.gov.hmcts.reform.sendgrid;
```

Comment:
I'm not sure I understand the reason of promoting these outside of the "civil" package.

The components we are developing, after all, are still part of the "civil" project. Wouldn't this choice risk introducing package conflicts or confuse people as to where these classes live?

Reply:
As these clients are not specific to Civil projects. so kept these outside. The reusabsle components which are specific to Civil only are going to be inside that package.